.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+# Description
+
+<!-- Please include a summary of the changes and the related issue. Also please include relevant motivation and context. List any dependencies that are required
+for this change. -->
+
+<!-- Remove the Below line if it doesn't related to Issues -->
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to
+      not work as expected)
+- [ ] This change requires a documentation update
+
+## How has this been tested?
```

Comment:
Please add a section: 

Have you self tested the UI and functionality? 

Answer would be no, or Yes